suites/nautilus/upgrades/upgrade_4.x-1_to_4.x.yaml

```diff
@@ -0,0 +1,100 @@
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+- test:
+    name: ceph ansible install rhcs 4.x cdn
+    polarion-id: CEPH-83571467
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4 
+        ceph_stable_release: nautilus
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_docker_registry_auth: false
+        dashboard_enabled: false
+        copy_admin_key: True
+        ceph_conf_overrides:
+          global:
+            osd_pool_default_pg_num: 128
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 128
+            mon_max_pg_per_osd: 4096
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster 4.x-1 cdn setup using ceph-ansible
+    abort-on-fail: True
+- test:
+    name: Upgrade ceph ansible to 4.x
+    polarion-id: CEPH-11110
```

Comment:
can we consider this TC automated? without following all the steps in it?

Reply:
As per the last discussion with @rakeshgm and other team members, it seems this is the only generic upgrade TC available. So I have added this, though all steps are not covered.

Reply:
@Ameenasuhani it'll update in polarion as run when run thru ci pipeline

Reply:
@XDsubhash I have created a test case. I will be adding this as polarion id
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573508

Reply:
thanks